src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/ItemAutomationPeer.cs

```diff
@@ -436,6 +436,18 @@ protected override bool IsEnabledCore()
             return false;
         }
 
```

Comment:
WindowAutomationPeer needs to override IsDialogCore(), to return true if the owning window is a dialog.  (Apologies if that's here somewhere - I haven't seen it yet.)

Reply:
Done.